docs/guides/references/configuration.mdx

```diff
@@ -682,6 +684,47 @@ This function was added in Cypress version `10.0.0` to replace the deprecated
 
 See the [plugins guide](/guides/tooling/plugins-guide) for more information.
 
+### stripCspDirectives
+
+Cypress will (by default) strip all CSP headers from the response before it is
+sent to the browser. This option allows for more granular control over which CSP
+directives are stripped from the CSP response headers, allowing you to test your
+application with CSP enabled.
+
+#### Strip All CSP Headers
+
+The default value of `stripCspDirectives` is `"all"`. This will remove all CSP
+headers from the response before it is sent to the browser. This option should
+be used if you do not depend on CSP for any tests in your application.
+
+#### Strip Minimum CSP Directives
+
+If you need to test your application with CSP enabled, you can set the
+`stripCspDirectives` option to `"minimum"`. This will allow all CSP headers to
+be sent to the browser _*except*_ those that would prevent Cypress from
+functioning Normally. The following CSP directives are automatically stripped
```

Comment:
Is "Normally" intentionally capitalized here?

Reply:
This is outdated. Sorry!

Reply:
...no...nor will it be for long 😉